docs/build_and_load.md

```diff
@@ -160,13 +161,18 @@ Instead of setting this up manually, we can use the
 run, use the following command:
 
 ```shell
-phg setup-environment --env-file phg_environment.yml
+phg setup-environment
 ```
 
-This command takes one parameter, `--env-file`. This is a path to the 
-Conda environment file. You can create the file 
-from scratch (`phg_environment.yml`)  and copy over the contents in 
-the following block:
+By using the prior default example, the `setup-environment` command 
+will extract an internal Conda environment file with the necessary 
+libraries and programs (_see following code block for example file_). 
+_Optionally_, if you want to add additional libraries and programs 
+to your PHGv2 Conda environment, you may specify the _optional_ 
+parameter, `--env-file`. This will be a path to a _local_ Conda 
+environment file. You can create the file from scratch (for example, 
+I will call mine `phg_environment.yml`) and copy over the contents 
+in the following block:
 
```

Comment:
Should you emphasize that when creating the environment file from scratch, the "name" must remain phgv2-conda for out internal processes to work?

Reply:
I do mention that in a note block a little further down in the file:

https://github.com/maize-genetics/phg_v2/blob/dc7189cc4e9574ae98224946d91717c0bbbaffd2/docs/build_and_load.md?plain=1#L214-L217

Reply:
Thanks - I thought it was in there somewhere.